src/Symfony/Component/Workflow/DefinitionBuilder.php

```diff
@@ -54,20 +54,36 @@ public function clear()
     {
         $this->places = [];
         $this->transitions = [];
-        $this->initialPlace = null;
+        $this->initialPlaces = null;
         $this->metadataStore = null;
 
         return $this;
     }
 
     /**
+     * @deprecated since Symfony 4.3. Use the setInitialPlaces() instead.
+     *
      * @param string $place
      *
      * @return $this
      */
     public function setInitialPlace($place)
     {
-        $this->initialPlace = $place;
+        @trigger_error(sprintf('Calling %s::setInitialPlace() is deprecated. Call %s::setInitialPlaces() instead.', __CLASS__, __CLASS__));
```

Comment:
missing "since Symfony 4.3" etc - I'd suggest removing the repeated FQCN

Reply:
It's already in the annotation. and about the FQCN, I took the same message from https://github.com/symfony/symfony/blob/4.3/src/Symfony/Component/Workflow/Definition.php#L50-L64

Reply:
That's a must: deprecation reports must be crystal clear, all runtime deprecations should report the affected package, Symfony or not. If some don't, they should be fixed.

Reply:
Fixed. I also fixed it in the definition class. I checked, everywhere else in the component it's OK.
I removed the double FQCN in both class